bedwars-plugin/src/main/java/com/tomkeuper/bedwars/configuration/MainConfig.java

```diff
@@ -78,6 +78,7 @@ public MainConfig(Plugin plugin, String name) {
         yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_LEAVE_DELAY, 3);
         yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_REJOIN_TIME, 60 * 5);
         yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_RE_SPAWN_INVULNERABILITY, 4000);
+        yml.addDefault(ConfigPath.GENERAL_CONFIGURATION_ANNOUNCEMENT_SEND_EVERY, 6000);
```

Comment:
how much time is this 6000? every minute? 30 seconds? 2 minutes?

Reply:
> how much time is this 6000? every minute? 30 seconds? 2 minutes?

5 minutes

Reply:
maybe set this to 300 and add in the taskTimer a `*20`

Reply:
> maybe set this to 300 and add in the taskTimer a `*20`

will change

Reply:
ok thanks